res/build.sh

```diff
@@ -28,8 +28,28 @@ sed -e "s,@PREFIX@,$PREFIX," res/void-docs.in > res/void-docs
 sed -e "s,@PREFIX@,$PREFIX," res/void-docs.1.in > res/void-docs.1
 
 # Build PDF
+
 echo "Building PDF"
+
+## Temporarily add OpenType version of Latin Modern to font cache.
+
+TMP_FONTDIR=$(mktemp -d -p ~/.fonts)
+mkdir -p $TMP_FONTDIR
```

Comment:
Well, the point of the `-p` flag is to create intervening directories if they don't already exist.

    TMP_FONTDIR=$(mktemp -d -p ~/.fonts)
 
creates a temporary directory using `~/.fonts` as its base, such as `~/.fonts/tmp.YHReGrbetL`. Then 
    
   mkdir -p $TMP_FONTDIR

will create that path, creating the `~/.fonts` directory in the process if necessary.

Did the above ordering fail on your system?

Reply:
It fails if `~/.fonts` doesn't exist, because it can't create the directory. Then `mktemp` does the job of creating the directory inside `.fonts`, so there's no need to call `mkdir` afterwards.

Reply:
Oh i see - i thought `mktemp` just created the directory name, but it actually creates the directory. Okay.